docker/config/tk/config_tool.py

```diff
@@ -0,0 +1,167 @@
+import customtkinter 
+from tkinter import *
+from tkinter import filedialog, StringVar
+import json
+import os
+
+data_file_path = os.path.join(os.getcwd(), 'data.json')
+
+#write input to terminal
+def save_config():
+    with open(data_file_path, "r") as jsonFile:
+        data = json.load(jsonFile)
+
+    data["Architecture"] = clicked.get()
+    data["Build tools"] = click_tools.get()
+    data["Directory"] = libchoice.cget("text")
+    data["Container prefix"] = entry.get()
+    data["libc"] = click_lib.get()
+
+
+    with open(data_file_path, "w") as jsonFile:
+        json.dump(data, jsonFile)
+
+    confirm()
+    
+def load_config():
+    jason = filedialog.askopenfile(filetypes=[("Json files", "*.json")])  # Open a dialog to select a directory
+    if jason:
+        data = json.load(jason)
+        config_map = {
+            'Architecture': clicked,
+            'Build tools': click_tools,
+            'libc': click_lib
+        }
+
+        for key, var in config_map.items():
+            value = data.get(key,'')
+            if value:
+                var.set(value)
+
+        directory = data.get("Directory", "")
+        if directory:
+            libchoice.config(text=directory)
+
+        
+        con_prefix = data.get("Container prefix", "")
+        if con_prefix:
+            entry.delete(0, END)  # Clear the entry first
+            entry.insert(0, con_prefix)
+
+        jason.close
+    
+
+    
+def select_dir():
+    directory = filedialog.askdirectory()  # Open a dialog to select a directory
+    if directory:
+        libchoice.configure(text=directory) 
+
+def confirm():
+    con_label = Label(app, text="Saved!", fg="green")
+    con_label.pack()
+
+    app.after(2000, con_label.destroy)
+
+# Dropdown menu options 
+options = [ 
+    "x86", 
+    "ARM", 
+    "MIPS", 
+    "PowerPc", 
+    "x64"
+]
+
+
+
+# build tools
+build_tools = [ 
+    "Build Tools", 
+    "Emualtion", 
+    "Both"
+]
+
+# libc choice
+libc_build = [ 
+    "glibc", 
+    "musl", 
+    "uclibc"
+]
+
+app = customtkinter.CTk()
+app.geometry("400x550")
```

Comment:
Consider storing these literals as constant variables at the top of the file, e.g.
DEF_WINDOW_SIZE = "400x550"
DEF_BG_COLOUR = "#000000"

Reply:
Changed to constants